images.qrc

```diff
@@ -100,5 +100,6 @@
         <file alias="banner.png">Images/banner.png</file>
         <file alias="icon.ico">Images/icon.ico</file>
         <file alias="transparent.png">Images/transparent.png</file>
+        <file alias="events.ey">Submodules/enigma-dev/events.ey</file>

```

Comment:
Maybe we should rename images.qrc or resources.qrc because the user will later replace icon packs dynamically. We don't want them to have to put `events.ey` into their icon pack.

Reply:
IRight, I was going to rename this file but didn't care to do it here

Reply:
I really wouldn't put it in with the images.
https://doc.qt.io/qt-5/resources.html#external-binary-resources